### DISCUSSION TITLE
EBYTE E32 Series

### DISCUSSION THREAD

**Discussion by Edgineer10** [sandeepmistry/arduino-LoRa#663] [Q&A]
May I ask if this Library also supports Ebyte E32-900T20D module since the module also have semtech's SX1276 chip?

**Comment by Kongduino** [comment]
E32-900T20D is a UART module. This liobrary won't work.

**Reply by Edgineer10** [reply]
Can this module communicate with Lilygo TTGO Lo32?

**Reply by Kongduino** [reply]
Yes. See https://github.com/LilyGO/TTGO-LoRa32-V2.1/blob/master/OLED_LoRa_Sender/OLED_LoRa_Sender.ino for instance.

**Reply by Edgineer10** [reply]
I appreciate your response. do you have any Idea on their configuration? Thank you so much in advance!

**Reply by Kongduino** [reply]
It's all in there in the code I linked. You have everything to set it up properly.

**Reply by Edgineer10** [reply]
I mean both the TTGO Lora32 and Ebyte E32, what should be done so that they could communicate with each other?
